### ISSUE TITLE
Peer dep warning when using fetch-mock 6.

### ISSUE THREAD

**Issue by tregusti** [gakimball/chai-fetch-mock#2]
```sh
yarn add -D fetch-mock # currently will install 6.0.0.beta.7
yarn add -D chai-fetch-mock # will give the warning below
```
> warning " > chai-fetch-mock@1.0.1" has incorrect peer dependency "fetch-mock@^5.1.x".

And of course this is since vthis module is not comaptible with v6. `fetchMock` is not a property of the mocked `fetch` anymore.

**Comment by gakimball** [comment]
Thanks for the heads up! I'll see about making time to get this library up to date, or I'd gladly take a PR if you're interested.

**Comment by gakimball** [comment]
Published `2.0.0` with this support. The fix was very simple, but fetch-mock 6.0 dropped support for Node < 7, so I bumped major versions.
